docs/release-notes/pytorch2.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**New Features**
+
+-  Environments: Add PyTorch2.0 Beta images.
+
+   -  We are providing a set of images with `pytorch==2.0.1`, `python==3.10.12`, and `CUDA==11.8`.
+      These images have not been fully tested against certain integrations, and are still in beta.
```

Comment:
this is a docs question but we have never used the notion of `beta` before. these things has always been branded as "experimental".

Reply:
I updated the phrasing, but I'll also wait for docs to review.